AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -249,7 +286,14 @@ - (void)startUpdate {
       return;
     }
     NSString *distributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
-    if (updateToken || distributionGroupId) {
+    @synchronized(self) {
+      if (self.updateFlowInProgress) {
+        MSLogDebug([MSDistribute logTag], @"Previous update flow is in progress. Ignore the request.");
+        return;
+      }
+      self.updateFlowInProgress = YES;
```

Comment:
Is that the right place to do this? It could also happen when resuming app from background and we don't want to check if already checking.